packages/react-next/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -161,6 +155,11 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>>
 
   protected renderSuggestions(): JSX.Element | null {
     const TypedSuggestionsControl = this.SuggestionsControlOfProperType;
+
+    if (this.props.suggestionItems) {
+      this.suggestionStore.updateSuggestions(this.props.suggestionItems!);
```

Comment:
Sorry if I asked this before, but is the cost of doing this every render a concern? Wondering if it would be better to do only in `componentDidUpdate` if `this.props !== prevProps` (any updates done there will be flushed before rendering, assuming you setState or forceUpdate).

Reply:
You did! And it shouldn't be an issue.